ansible_collections/arista/avd/roles/eos_designs/templates/facts/switch/switch.j2

```diff
@@ -397,3 +397,37 @@ switch:
 {# switch.vlans #}
   vlans: {{ leaf.vlans | unique }}
 {% endif %}
+
+{# EVPN multihoming configuration on VTEPs #}
+{# switch.link_tracking #}
+{% if switch_data.combined.link_tracking.enabled is arista.avd.defined(true) %}
+{%     if switch_data.combined.link_tracking.groups is arista.avd.defined %}
+  link_tracking_groups:
+{#         Do not sort the group as the first group will be the default group for the end-points #}
+{%         for lt_group in switch_data.combined.link_tracking.groups %}
+{%             if lt_group.name is arista.avd.defined %}
+    - name: {{ lt_group.name }}
+      recovery_delay: {{ lt_group.recovery_delay | arista.avd.default(500) }}
+{%                 if lt_group.links_minimum is arista.avd.defined %}
+      links_minimum: {{ lt_group.links_minimum }}
+{%                 endif %}
+{%             endif %}
+{%         endfor %}
+{%     else %}
+  link_tracking_groups:
+    - name: LT_GROUP1
+      recovery_delay: 500
+{%     endif %}
+{% endif %}
+
+{% if switch.vtep is arista.avd.defined(true) %}
+{#     switch.lacp_port_id #}
+{%     if switch_data.combined.lacp_port_id_range.enabled is arista.avd.defined(true) %}
+{%         set group_length = switch_data.node_group.nodes | arista.avd.default([inventory_hostname]) | length %}
+{%         set port_range = switch_data.combined.lacp_port_id_range.size | arista.avd.default(128) %}
```

Comment:
Any specific reason to choose 128 as a default? In my opinion, 100 is much more human-friendly, specially for troubleshooting. I also think 100 AA ports per device cover majority of scenarios

Reply:
These ports are not the switch ports. These are the lacp port-id. Recommended practice is to keep the lacp port-id range = # ports on the switch(not mandatory but considering the worst case). This doesn't allocate the AA ports on switch but allocates the port-id from the range(1-128), to those actual switch ports which are currently on LAG.

Page 16 of the design guide: 
![image](https://user-images.githubusercontent.com/88734745/151862720-893c115b-cdf0-434e-a309-feaf4e1a0e1a.png)